examples/basics/UI.py

```diff
@@ -0,0 +1,246 @@
+            ["From xml", "From recording"], label="Simulation Type", value="From xml"
+        )
+
+        with gr.Row():
+            with gr.Column(visible=True) as realtime_col:
+                # File input for live simulation
+                input_files = gr.File(
+                    label="Input Files (for From xml)", file_count="multiple"
+                )
+            with gr.Column(visible=False) as playback_col:
+                # File inputs for playback simulation
+                trajectory_files = gr.File(
+                    label="Trajectory Files (for playback)", file_count="multiple"
+                )
+                state_file = gr.File(
+                    label="State File (for playback)", file_count="multiple"
+                )
+
+        with gr.Row():
+            with gr.Column():
+                gr.Markdown("## Verbosity")
+                # Radio button for verbosity level
+                verbosity = gr.Radio(
+                    ["Normal", "Verbose", "Super verbose"],
+                    label="Verbosity Level",
+                    value="Normal",
+                )
+                show_progression = gr.Checkbox(label="Show simulation progression")
+
+                gr.Markdown("## Network")
+                # Textbox for server name
+                server_name = gr.Textbox(
+                    label="Server name", value="NanoVer-RS iMD Server"
+                )
+                # Number input for port
+                port = gr.Number(label="Port", value=38801)
+
+                gr.Markdown("## Simulation")
+                # Slider for simulation FPS
+                simulation_fps = gr.Slider(1, 60, value=30, label="Simulation FPS")
+                # Number inputs for frame and force intervals
+                frame_interval = gr.Number(label="Frame interval", value=5)
+                force_interval = gr.Number(label="Force interval", value=5)
+                # Checkboxes for simulation options
+                start_paused = gr.Checkbox(label="Start simulation paused")
+                include_velocities = gr.Checkbox(
+                    label="Include the velocities in the frames"
+                )
+                include_forces = gr.Checkbox(label="Include the forces in the frames")
+
+            with gr.Column():
+                gr.Markdown("## Recording")
+                with gr.Group():
+                    # Checkboxes and textboxes for recording options
+                    record_stats = gr.Checkbox(label="Record statistics")
+                    stats_file = gr.Textbox(label="Statistics file")
+                    stats_fps = gr.Number(label="Statistics FPS", value=4)
+                with gr.Group():
+                    record_trajectory = gr.Checkbox(label="Record trajectory")
+                    trajectory_output_file = gr.Textbox(
+                        label="Trajectory output file",
+                        value=f"{server_name.value}.traj",
```

Comment:
See comments on server name

Reply:
Do we want to automatically provide a default name for the output files? In the rust server there is no default name or path for the input files, so if we're looking for parity with this we may wish to remove the default names inserted here